docs/technical-documentation/rest-get.md

```diff
@@ -0,0 +1,480 @@
+
+For media you use `/media/{id}`
+
+&#x1F34E; For example: 
+
+```
+curl -u admin:islandora http://localhost:8000/media/2?_format=json
+
+{
+  "mid": [
+    {
+      "value": 2
+    }
+  ],
+  "uuid": [
+    {
+      "value": "e75e609d-510c-4764-9280-665bb7026161"
+    }
+  ],
+  "vid": [
+    {
+      "value": 2
+    }
+  ],
+  "langcode": [
+    {
+      "value": "en"
+    }
+  ],
+  "bundle": [
+    {
+      "target_id": "image",
+      "target_type": "media_type",
+      "target_uuid": "3860e653-201b-4509-89dd-628c446d81cb"
+    }
+  ],
+  "revision_created": [
+    {
+      "value": "2019-03-01T19:43:48+00:00",
+      "format": "Y-m-d\\TH:i:sP"
+    }
+  ],
+  "revision_user": [],
+  "revision_log_message": [],
+  "status": [
+    {
+      "value": true
+    }
+  ],
+  "name": [
+    {
+      "value": "1-Service File.jpg"
+    }
+  ],
+  "thumbnail": [
+    {
+      "target_id": 3,
+      "alt": null,
+      "title": null,
+      "width": 800,
+      "height": 1333,
+      "target_type": "file",
+      "target_uuid": "e97ad038-dcb3-4d81-aa91-c945b2fe092c",
+      "url": "http:\\/\\/localhost:8000\\/sites\\/default\\/files\\/2019-03\\/1-Service%20File.jpg"
+    }
+  ],
+  "uid": [
+    {
+      "target_id": 1,
+      "target_type": "user",
+      "target_uuid": "46a47057-de2d-4ce2-87ae-dbe4551209b8",
+      "url": "\\/user\\/1"
+    }
+  ],
+  "created": [
+    {
+      "value": "2019-03-01T19:43:48+00:00",
+      "format": "Y-m-d\\TH:i:sP"
+    }
+  ],
+  "changed": [
+    {
+      "value": "2019-03-01T19:43:48+00:00",
+      "format": "Y-m-d\\TH:i:sP"
+    }
+  ],
+  "default_langcode": [
+    {
+      "value": true
+    }
+  ],
+  "revision_translation_affected": [
+    {
+      "value": true
+    }
+  ],
+  "content_translation_source": [
+    {
+      "value": "und"
+    }
+  ],
+  "content_translation_outdated": [
+    {
+      "value": false
+    }
+  ],
+  "field_access_terms": [],
+  "field_file_size": [
+    {
+      "value": 166608
+    }
+  ],
+  "field_height": [
+    {
+      "value": 1333
+    }
+  ],
+  "field_media_image": [
+    {
+      "target_id": 3,
+      "alt": null,
+      "title": null,
+      "width": 800,
+      "height": 1333,
+      "target_type": "file",
+      "target_uuid": "e97ad038-dcb3-4d81-aa91-c945b2fe092c",
+      "url": "http:\\/\\/localhost:8000\\/sites\\/default\\/files\\/2019-03\\/1-Service%20File.jpg"
+    }
+  ],
+  "field_media_of": [
+    {
+      "target_id": 1,
+      "target_type": "node",
+      "target_uuid": "8322e36e-f8ec-4fd9-919d-52aed7b17a52",
+      "url": "\\/node\\/1"
+    }
+  ],
+  "field_media_use": [
+    {
+      "target_id": 18,
+      "target_type": "taxonomy_term",
+      "target_uuid": "e8c698a7-8c15-47c4-8567-1c27f8f0c19e",
+      "url": "\\/taxonomy\\/term\\/18"
+    }
+  ],
+  "field_mime_type": [
+    {
+      "value": "image\\/jpeg"
+    }
+  ],
+  "field_width": [
+    {
+      "value": 800
+    }
+  ]
+}
+```
+
```

Comment:
It is useful to know which nodes a media is associated with, since this defines the "parent/child" relationship. Can we add some text here such as:

> `field_media_of` indicates which node(s) the media is associated with. The node can be referenced using the value of the `target_id` key (in this example, 1) or the `target_uuid` key (8322e36e-f8ec-4fd9-919d-52aed7b17a52).